src/main/java/jalasoft/com/utils/TaskBuilder.java

```diff
@@ -0,0 +1,32 @@
+package jalasoft.com.utils;
+
+import javax.servlet.http.HttpServletRequest;
+
+import jalasoft.com.models.CustomDate;
+import jalasoft.com.models.Status;
+import jalasoft.com.models.Task;
+
+public class TaskBuilder {
+

```

Comment:
same than in ToDoListPersister , why all the methods of this class are static? if it is a utility class, you should prevent the creation of objects of this class